### ISSUE TITLE
Can't register if no family name is set

### ISSUE THREAD

**Issue by felix-schindler** [denoland/meet-me#64]
I did not set a family name in my Google account. When I try to log in I get the following error:

```json
{
   "code": "invalid-argument",
   "name": "FirebaseError",
   "status": 500,
   "message": "Function setDoc() called with invalid data. Unsupported field value: undefined (found in field familyName in document users/{someId})",
   "stack": "FirebaseError: Function setDoc() called with invalid data. Unsupported field value: undefined (found in field familyName in document users/{someId})"
}
```

**Comment by kt3k** [comment]
Thanks for your report! We tried to fix this in #65 . Could you try signing in again?

**Comment by felix-schindler** [comment]
Works perfectly fine now, thanks!
